set_inputfiles.yaml

```diff
@@ -76,6 +76,77 @@
+    DY: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016preVFPUL/DYJetsToLL_2016preVFPUL_ee.root"
+
+"2016postVFPUL":
+  embeddingselection:
+    folder: "ntuple"
+    Data: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016postVFPUL/DoubleMuon_2016postVFPUL.root"
+    # DataF: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016postVFPUL/DoubleMuon_Run2016F_2016postVFPUL.root"
+    # DataG: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016postVFPUL/DoubleMuon_Run2016G_2016postVFPUL.root"
+    # DataH: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016postVFPUL/DoubleMuon_Run2016H_2016postVFPUL.root"
+
+  muon:
+    folder: "ntuple"
+    Data: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016postVFPUL/SingleMuon_2016postVFPUL.root"
+    # Embedding: "/ceph/sbrommer/embedding_ul/scalefactors/inputfiles/2018_v6/ntuple_MuEmbedding_UL2018.root"
+    DY: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016postVFPUL/DYJetsToLL_2016postVFPUL_mm.root"
+
+  electron:
+    folder: "ntuple"
+    Data: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016postVFPUL/SingleElectron_2016postVFPUL.root"
+    # Embedding: "/ceph/sbrommer/embedding_ul/scalefactors/inputfiles/2018_v6/ntuple_ElEmbedding_UL2018.root"
+    DY: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2016postVFPUL/DYJetsToLL_2016postVFPUL_ee.root"
+
+"2017UL":
+  embeddingselection:
+    folder: "ntuple"
+    Data: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2023_03_17_TandP/2017UL/DoubleMuon_2017UL.root"
+    # DataB: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/DoubleMuon_Run2017B_2017UL.root"
+    # DataC: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/DoubleMuon_Run2017C_2017UL.root"
+    # DataD: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/DoubleMuon_Run2017D_2017UL.root"
+    # DataE: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/DoubleMuon_Run2017E_2017UL.root"
+    # DataF: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/DoubleMuon_Run2017F_2017UL.root"
+
+  muon:
+    folder: "ntuple"
+    # Data: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/SingleMuon_2017UL.root"
+    # Embedding: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/MuonEmbedding_2017UL.root"
+    # DY: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/DYJetsToLL_2017UL_mm.root"
+    Data: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2023_03_17_TandP/2017UL/SingleMuon_2017UL.root"
+    Embedding: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2023_03_17_TandP/2017UL/MuonEmbedding_2017UL.root"
+    DY: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2023_03_17_TandP/2017UL/DYJetsToLL_2017UL_mm.root"
```

Comment:
should i copy these files to my folders ?

Reply:
Yes, could make sense to do so.